app/views/boards/_board.html.erb

```diff
@@ -1,6 +1,17 @@
-<tr id="<%= dom_id board %>">
-  <td><%= board.name %></td>
-  <td><%= link_to 'Show', board %></td>
-  <td><%= link_to 'Edit', edit_board_path(board) %></td>
-  <td><%= link_to 'Destroy', board, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-</tr>
+<div class='flex flex-col border px-4 py-2 bg-rs-gray-100 rounded shadow xl:w-52 xl:h-28 justify-self-center' id="<%= dom_id board %>">
```

Comment:
This line it's a little bit long (136)

What do you think about this change?
```suggestion
<div 
  class='flex flex-col border px-4 py-2 bg-rs-gray-100 rounded shadow xl:w-52 xl:h-28 justify-self-center' 
  id="<%= dom_id board %>"
>
```

Reply:
Done!